addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="web.ControlPanel" owl="1">
         <div class="o_control_panel d-flex flex-column gap-3 gap-lg-1 px-3 pt-2 pb-3" t-ref="root">
             <div class="o_control_panel_main d-flex flex-wrap flex-lg-nowrap justify-content-between align-items-lg-start gap-3 flex-grow-1">
-                <div class="o_control_panel_breadcrumbs d-flex align-items-center gap-1 order-0">
+                <div class="o_control_panel_breadcrumbs d-flex align-items-center gap-1 order-0 h-100">
```

Comment:
This solves timesheet but in general it shouldn't be needed.
We'll have to investigate further, ok for now 👍